src/routes/safe/components/Transactions/TxList/QueueTxList.tsx

```diff
@@ -79,7 +82,13 @@ type QueueTxListProps = {
 
 export const QueueTxList = ({ transactions }: QueueTxListProps): ReactElement => {
   const { txLocation } = useContext(TxLocationContext)
-  const title = txLocation === 'queued.next' ? 'NEXT TRANSACTION' : 'QUEUE'
+  const nonce = useSelector(safeNonceSelector)
+
+  const getTitle = () => {
+    return txLocation === 'queued.next'
+      ? 'NEXT TRANSACTION'
+      : `QUEUE - Transaction with nonce ${nonce} needs to be executed next`
+  }
```

Comment:
Is a function really needed? We could just define the `txLocation` var in the outer scope.

Reply:
totally